proteus/mprans/SpatialTools.py

```diff
@@ -83,7 +84,7 @@ def _attachAuxiliaryVariable(self, key):
         """
         if key not in self.auxiliaryVariables:
             if key == 'RigidBody':
-                self.auxiliaryVariables[key] = True
+                self.auxiliaryVariables[key] = bd.RigidBody(self)
```

Comment:
Perhaps it would be better to have Shapes hold default values of `None` for those variables.  If RigidBody's are not used, then it doesn't seem to have a need for them - and if they are, the user then would be forced to tell the Shape the parameters they have (and then Shape would pass them to BodyDynamics here).

Or, of course, a method for passing the values directly to BodyDynamics - or pulling them out of Context (but while `dt_init` is one of the most consistent variables in the tests I've seen, I'm wary of constantly pulling things out of Context and hoping the user understands that they needed those particular variables to be there in the first place - and `cfl_target`might be even more obscure [especially if it doesn't have to necessarily be proportional to `runCFL`] ).  

Alternatively, this might suggest that time stepping / CFL parameters might deserve their own class as per [Tristan's design here](https://github.com/erdc-cm/proteus/issues/428#issuecomment-230259668), allowing us to pull them out of the hat with a little less mystery and obscurity than just plain variables in Context.

In any case, it's useful to know which access method is going to get used in order to anticipate the changes needed to fit it in.